geosyspy/services/analytics_processor_service.py

```diff
@@ -0,0 +1,567 @@
+            },
+            "data":
+                [
+                    {
+                        "id": "SeasonField:" + seasonfield_id + "@ID",
+                        "crop": crop,
+                        "sowing_date": sowing_date,
+                        "geom": geometry
+                    }
+                ]
+        }
+
+        processor_endpoint: str = urljoin(self.base_url,
+                                          GeosysApiEndpoints.LAUNCH_PROCESSOR_ENDPOINT.value.format(api_processor_path))
+
+        response = self.http_client.post(processor_endpoint, payload)
+
+        if response.ok:
+            task_id = json.loads(response.content)["taskId"]
+            return task_id
+        else:
+            logging.info(response.status_code)
+            raise ValueError(response.content)
+
+    def launch_brazil_in_season_crop_id_processor(self,
+                                                  start_date: str,
+                                                  end_date: str,
+                                                  season: str,
+                                                  seasonfield_id: str,
+                                                  geometry: str):
+        """launch a brazil-in-season-crop-id analytics processor and get the task id in result
+
+            Args:
+                start_date (str) : the start date used for the request (format YYYY-MM-dd)
+                end_date (str) : the end date used for the request (format YYYY-MM-dd)
+
+                season (str): the season name,
+                seasonfield_id (sfd) : seasonField geosys uniqueId
+                geometry (str): the geometry to calculate the analytic (WKT or GeoJSON),
+
+            Returns:
+                taskId (str)
+        """
+        parameters_profile = "brazil_in_season_crop_id_default"
+        api_processor_path = "brazil_in_season_crop_id"
+
+        # build payload for api call
+        payload = {
+            "parametersProfile": parameters_profile,
+            "parameters": {
+                "start_date": start_date,
+                "end_date": end_date,
+                "season": season
+            },
+            "data":
+                [
+                    {
+                        "id": "SeasonField:" + seasonfield_id + "@ID",
+                        "geom": geometry
+                    }
+                ]
+        }
+
+        processor_endpoint: str = urljoin(self.base_url,
+                                          GeosysApiEndpoints.LAUNCH_PROCESSOR_ENDPOINT.value.format(api_processor_path))
+
+        response = self.http_client.post(processor_endpoint, payload)
+
+        if response.ok:
+            task_id = json.loads(response.content)["taskId"]
+            return task_id
+        else:
+            logging.info(response.status_code)
+            raise ValueError(response.content)
+
+
+    def launch_potential_score_processor(self,
```

Comment:
launch_potential_score_processor() method is declared twice with the same signature